backend/campaigns/migrations/0002_marketing_group.py

```
@@ -0,0 +1,49 @@
+    group, _ = Group.objects.get_or_create(name=MARKETING_GROUP_NAME)
+    for model, actions in MARKETING_PERMISSIONS:
+        for action in actions:
+            permission = Permission.objects.filter(
+                content_type__app_label='campaigns',
+                codename=f'{action}_{model}',
+            ).first()
+            if permission:
+                group.permissions.add(permission)
+
+
+def remove_marketing_group(apps, schema_editor):
```

Ruff (0.16.0)

[warning] Unused function argument: `schema_editor`

(ARG001)